configure

```diff
@@ -40,11 +40,30 @@ if (config === 'dev') {
   browserify += ' -g uglifyify'
 }
 
+function isSubmitDependency(name) {
+  const dependencies = [
+    'github-url-to-object',
+    'html-to-react',
+    'url',
+    'marky-markdown',
+    'lodash.camelcase',
+    'immutable',
+    'path'
+  ]
+
+  return dependencies.some(dependency => dependency === name)
+}
+
 const dependencies = Object.keys(require('./package.json').dependencies).filter(
   name => name !== 'semantic-ui-css'
 )
 const excludes = `-x ${dependencies.join(' -x ')}`
-const requires = `-r ${dependencies.join(' -r ')}`
+const requires = `-r ${dependencies
+  .filter(name => !isSubmitDependency(name))
+  .join(' -r ')}`
+const submit = `-r ${dependencies
```

Comment:
Need a more descriptive name. `submitRequires`? But maybe we don't actually need this at all. See review description.